### PR TITLE
Merge styles into gr-test-result-table-module module

### PR DESCRIPTION
**Pull request by paladox**
This will give it some Polymer 2 compatibility.

### CODE REVIEW

**Comment by paladox** [comment]
Not fully compat yet, will need undefined checks which i can add later.

File: test_result-0.1.html [hunk 1]
```diff
@@ -59,11 +56,6 @@
           font-weight: bold;
       }
     </style>
-  </template>
-</dom-module>
-
-<dom-module id="gr-test-result-table-module">
-  <template>
     <style include="gr-test-result-table-style-module"></style>
```
Comment (dburm) [inline test_result-0.1.html]: Remove style inclusion as well

Comment (paladox) [reply test_result-0.1.html]: Ah, i missed that, thanks!

**Comment by paladox** [comment]
Thanks!

**Pull request merged**